app/questionnaire/navigator.py

```diff
@@ -54,7 +51,7 @@ def build_path(self, blocks, this_location):
 
         # Keep going unless we've hit the last block
         while block_index < blocks_len:
-            if this_location['block_id'] in self.CLOSING_INTERSTITIAL_PATH:
+            if this_location.block_id in self.CLOSING_INTERSTITIAL_PATH:
```

Comment:
maybe this should also be in the Location class as well (if something's opening/closing interstitial?)